src/Microsoft.Extensions.Configuration.AzureAppConfiguration/JsonConfiguration/JsonConfigurationParser.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.JsonConfiguration
+{
+    internal static class JsonConfigurationParser
```

Comment:
>JsonConfigurationParser [](start = 26, length = 23)

Would it make sense to move the functionality of this type into JsonKeyValueAdapter? Parse looks like it should be the implementation for `JsonKeyValueAdapter.ProcessKeyValue` and IsJsonContentType could probably be a private method.

Reply:
On second thought `IsJsonContentType ` could just be inlined into CanProcess.